tests/Aspire.RabbitMQ.Client.Tests/AspireRabbitMQLoggingTests.cs

```diff
@@ -148,6 +153,38 @@ public void TestException()
         Assert.True(string.IsNullOrEmpty(errorEvent[4].Value?.ToString()));
     }
 
+#if !RABBITMQ_V6
+    private static readonly object s_log =
+        Type.GetType("RabbitMQ.Client.Logging.RabbitMqClientEventSource, RabbitMQ.Client")!
+            .GetField("Log", BindingFlags.Static | BindingFlags.Public)!
+            .GetValue(null)!;
+#endif
+
+    private static void LogInfo(string message)
+    {
+#if RABBITMQ_V6
+        RabbitMqClientEventSource.Log.Info(message);
+#else
+        s_log.GetType().GetMethod("Info")!.Invoke(s_log, new object[] { message });
```

Comment:
Is it worth caching these methods, or is this nearly equivalently fast these days? @steveharter

Reply:
These are methods used in tests that get called once. Theres no need to cache them.